lib/src/Pos/Client/CLI/NodeOptions.hs

```diff
@@ -13,33 +13,31 @@ module Pos.Client.CLI.NodeOptions
        , usageExample
        ) where
 
-import           Universum hiding (show)
-
-import           Data.Version (showVersion)
-import           NeatInterpolation (text)
-import           Options.Applicative (Parser, auto, execParser, footerDoc, fullDesc, header, help,
-                                      helper, info, infoOption, long, metavar, option, progDesc,
-                                      strOption, switch, value)
-import           Prelude (show)
+import           Universum                    hiding (show)
+
+import           Data.Version                 (showVersion)
+import           NeatInterpolation            (text)
+import           Options.Applicative          (Parser, auto, execParser, footerDoc, fullDesc,
+                                               header, help, helper, info, infoOption, long,
+                                               metavar, option, progDesc, strOption, switch, value)
 import           Text.PrettyPrint.ANSI.Leijen (Doc)
 
-import           Paths_cardano_sl (version)
+import           Paths_cardano_sl             (version)
 
-import           Pos.Client.CLI.Options (CommonArgs (..), commonArgsParser, optionalJSONPath)
-import           Pos.HealthCheck.Route53 (route53HealthCheckOption)
-import           Pos.Network.CLI (NetworkConfigOpts, networkConfigOption)
-import           Pos.Statistics (EkgParams, StatsdParams, ekgParamsOption, statsdParamsOption)
-import           Pos.Util.BackupPhrase (BackupPhrase, backupPhraseWordsNum)
-import           Pos.Util.CompileInfo (CompileTimeInfo (..), HasCompileInfo, compileInfo)
-import           Pos.Util.TimeWarp (NetworkAddress)
+import           Pos.Client.CLI.Options       (CommonArgs (..), commonArgsParser, optionalJSONPath)
+import           Pos.HealthCheck.Route53      (route53HealthCheckOption)
+import           Pos.Network.CLI              (NetworkConfigOpts, networkConfigOption)
+import           Pos.Statistics               (EkgParams, StatsdParams, ekgParamsOption,
+                                               statsdParamsOption)
+import           Pos.Util.CompileInfo         (CompileTimeInfo (..), HasCompileInfo, compileInfo)
+import           Pos.Util.TimeWarp            (NetworkAddress)
```

Comment:
It looks like `stylish-haskell` picked wrong configuration (the old one).

Reply:
Yes, I have been wondering about this for a while, and I have still to understand why, considering the correct stylish-haskell is now committed to the main repo. I will have to double-check.